package.json

```diff
@@ -132,12 +135,23 @@
 		},
 		"verbose": true
 	},
+	"lint-staged": {
+		"*.js": [
+			"prettier --write",
+			"eslint --fix",
+			"git add"
+		]
+	},
 	"scripts": {
 		"prebuild": "check-node-version --package",
 		"build": "cross-env BABEL_ENV=default NODE_ENV=production webpack",
 		"gettext-strings": "cross-env BABEL_ENV=gettext webpack",
 		"lint": "eslint .",
+		"lint:fix": "eslint . --fix",
```

Comment:
If nothing else, it would be good to get this command into master 🙂

Reply:
Added in https://github.com/WordPress/gutenberg/pull/4712